minecraft_servers/hardwarecraft/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "server_name": "Hardware Craft",
```

Comment:
Please change the `server_name` to `hardwarecraft` as it must be the same as the directory name.